paasta_tools/api/views/service.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+PaaSTA service.
+"""
+from pyramid.view import view_config
+
+from paasta_tools.utils import list_all_instances_for_service
+from paasta_tools.utils import load_system_paasta_config
+
+
+@view_config(route_name='service.list', request_method='GET', renderer='json')
+def list_instances(request):
+    service = request.matchdict['service']
+    cluster = load_system_paasta_config().get_cluster()
+    instances = list_all_instances_for_service(service, clusters=[cluster])
+    return {'instances': ','.join(instances)}
```

Comment:
this should be an array that get's processed by the json renderer, not a comma-separated string